lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -273,9 +273,6 @@ def invalid_v1_location(package)
         def invalid_location_uri_error(package, location_options)
           path = location_options.url
           return nil if path == ZendeskAppsSupport::Manifest::LEGACY_URI_STUB
-          if path.include?('{{setting.')
-            return location_options.signed ? ValidationError.new(:signed_setting_uri, uri: path) : nil
```

Comment:
Hang on, we shouldn't remove the whole block `if path.include?('{{setting.')` but should simply `return nil` instead right? Otherwise, it would continue to the check below where it tries to parse the path and check for absolute/relative url, which is not expected.

Reply:
Good catch! Discussed in 🍖, it may break the app if we don't return `nil` here. I have added this condition back now with `return nil`.

Reply:
Actually, would make sense to add one test for this scenario.